katz/src/main/kotlin/katz/instances/IorMonad.kt

```diff
@@ -16,12 +16,12 @@
 
 package katz
 
-class IorMonad<A>(val semigroup: Semigroup<A>) : Monad<HK<Ior.F, A>> {
-    override fun <B> pure(b: B): HK2<Ior.F, A, B> =
-            Ior.Right(b)
+class IorMonad<L>(val AA: Semigroup<L>) : Monad<HK<Ior.F, L>> {
+    override fun <A, B> flatMap(fa: HK<HK<Ior.F, L>, A>, f: (A) -> HK<HK<Ior.F, L>, B>): HK<HK<Ior.F, L>, B> =
```

Comment:
we could create an issue to change that we have to return concrete types and create and use typealias like in Coproduct